src/App/App.tsx

```diff
@@ -1,14 +1,27 @@
-import { BrowserRouter as Router } from "react-router-dom";
+import { Suspense, lazy } from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import { ErrorBoundary } from "react-error-boundary";
 import { GlobalStyle, Wrapper } from "./App.styles";
 import Header from "../components/Header";
+import SuspenseFallback from "../components/SuspenseFallback";
+import ErrorBoundaryFallback from "../components/ErrorBoundaryFallback";
+
+const Home = lazy(() => import("../pages/Home"));
 
 function App() {
   return (
     <Wrapper>
-      <Router>
-        <GlobalStyle />
-        <Header />
-      </Router>
+      <ErrorBoundary FallbackComponent={ErrorBoundaryFallback}>
```

Comment:
duży plus za to! Warto też mieć na uwadze taki scenariusz (w przypadku złożonych aplikacji gdzie jest przetwarzane sporo danych), że Error Boundary może być dodany dla konkretnego modułu a nie całej aplikacji. Spowoduje to, że w przypadku błędu wyłączony z użycia będzie tylko dany moduł, gdzie błąd wystąpił a nie cała apka, tutaj więcej:
https://reactjs.org/docs/error-boundaries.html#where-to-place-error-boundaries

Reply:
Oczywiście w przypadku tej apki jest super, jeden boundary będzie git :D